agent.go

```diff
@@ -0,0 +1,215 @@
+package main
+
+import (
+	"os"
+	"path/filepath"
+)
+
+// initImportRules initializes and returns the import rules map (source -> target)
+// Import reads from agent-specific locations and writes to Default agent locations
+func initImportRules() (map[Agent][]RulePath, error) {
+	homeDir, err := os.UserHomeDir()
+	if err != nil {
+		return nil, err
+	}
+
+	importRules := make(map[Agent][]RulePath)
```

Comment:
This func could be much simpler. Just a map from source to target path.
```suggestion
	importRules := make(map[string]string)
```